pkg/controller/install/install_controller.go

```diff
@@ -108,11 +109,25 @@ func (r *ReconcileInstall) Reconcile(request reconcile.Request) (reconcile.Resul
 	return reconcile.Result{}, nil
 }
 
+func ResourceScopeFilter(scope servingv1alpha1.InstallationScope) mf.Transformer {
```

Comment:
I'm comfortable with this for now, but we will need to go back over the exact required permissions.